src/test/java/tech/jhipster/lite/cli/command/infrastructure/primary/JHLiteCommandTest.java

```diff
@@ -39,16 +61,154 @@ void shouldShowHelpMessageWhenNoCommand(CapturedOutput output) {
     );
   }
 
-  @Test
-  void shouldListModules(CapturedOutput output) {
-    String[] args = { "list" };
-    CommandLine cmd = new CommandLine(jhliteCommand, factory);
+  @Nested
+  @DisplayName("list")
+  class ListModulesCommandTest {
```

Comment:
There's no need to use `Test` suffix for nested classes:
```suggestion
  class ListModules {
```